CBLClient/Apps/CBLTestServer-Dotnet/TestServer.UWP/run.ps1

```diff
@@ -2,8 +2,8 @@ param (
     [Parameter()][string]$AppDirectory
 )
 
-$PackageFullName="b0bd6c4c-e257-44c1-aeeb-7188f65af9d0_1.0.0.0_x64__1v3rwxh47wwxj"
-$LaunchName="b0bd6c4c-e257-44c1-aeeb-7188f65af9d0_1v3rwxh47wwxj!App"
+$PackageFullName="b0bd6c4c-e257-44c1-aeeb-7188f65af9d0_1.0.0.0_x64__75cr2b68sm664"
+$LaunchName="b0bd6c4c-e257-44c1-aeeb-7188f65af9d0_75cr2b68sm664!App"
```

Comment:
where are the hash value coming from? how are they generated?

Reply:
The has is generated by project and when we generate certificate. We don't know the way to generate the has programatically. Even Jim didn't have a good way to do it. So every time we renew certs we need to change the directory. If you have a easier way to find the package name, let me know